src/Components/Facility/FacilityCreate.tsx

```diff
@@ -682,339 +670,350 @@ export const FacilityCreate = (props: FacilityProps) => {
     case 3:
       return (
         <div className="px-2 pb-2">
```

Comment:
Remove the `div` that's present before `Page`. The page component will add consistent padding for all pages.